src/olympia/landfill/serializers.py

```diff
@@ -316,7 +316,7 @@ def create_featured_theme(self):
             promoted=RECOMMENDED,
         )
         addon.save()
-        generate_collection(addon, app=FIREFOX, type=amo.COLLECTION_RECOMMENDED)
+        generate_collection(addon, app=FIREFOX)
         print('Created Theme {0} for testing successfully'.format(addon.name))
 
     def create_featured_collections(self):
```

Comment:
Do we still need this? Are featured collections still a thing?

Reply:
They aren't.  I didn't touch landfill more than I had to... though that's been our strategy for years so it's an unmaintained mess.  I'll delete this function - it's likely a lot of the other code in here that generates collections is unnecessary too.

Reply:
https://github.com/mozilla/addons-server/pull/16963/commits/f4160ce7401b10b606fc725c0ed582a84855756a